state/Cargo.toml

```
@@ -0,0 +1,16 @@
+[package]
+name = "escrow-state"
+version = "0.1.0"
+edition = "2021"
+license = "MIT"
+authors = ["Gear Technologies"]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+gstd = { git = "https://github.com/gear-tech/gear" }
```

Should we use `stable` gstd?
```suggestion
gstd = { git = "https://github.com/gear-tech/gear", branch = "stable" }
```